manager/notifier.go

```diff
@@ -392,6 +456,14 @@ func (n *notifier) handleNotification(a *Alert, op notificationOp, config *pb.No
 			glog.Error("Error sending HipChat notification: ", err)
 		}
 	}
+	for _, scConfig := range config.SlackConfig {
+		if op == notificationOpResolve && !scConfig.GetSendResolved() {
+			continue
+		}
+		if err := n.sendSlackNotification(op, scConfig, a); err != nil {
+			glog.Error("Error sending Slack notification: ", err)
```

Comment:
`Errorln` would be more idiomatic here.
Maybe you could change that in the calls above as well.